target/linux/lantiq/xway/base-files/etc/board.d/02_network

```diff
@@ -0,0 +1,126 @@
+#!/bin/sh
+#
+# Copyright (C) 2011-2015 OpenWrt.org
+#
+
+. /lib/functions/uci-defaults.sh
+. /lib/functions/system.sh
+
+lantiq_setup_interfaces()
+{
+	local board="$1"
+
+	case "$board" in
+	arcadyan,arv4510pw)
+		ucidef_add_switch "switch0" \
+			"0:lan:4" "2:lan:2" "1:lan:3" "3:lan:1" "5t@eth0"
+		;;
+	arcadyan,arv4519pw|\
+	arcadyan,arv7510pw22|\
+	arcadyan,arv7518pw|\
+	arcadyan,arv752dpw22|\
+	arcadyan,arv8539pw22|\
+	buffalo,wbmr-hp-g300h)
+		ucidef_add_switch "switch0" \
+			"0t@eth0" "2:lan" "3:lan" "4:lan" "5:lan"
+		;;
+	arcadyan,arv7506pw11|\
+	audiocodes,mp-252|\
+	siemens,gigaset-sx76x)
+		ucidef_add_switch "switch0" \
+			"1:lan:4" "2:lan:3" "3:lan:2" "4:lan:1" "5t@eth0"
+		;;
+	arcadyan,arv7519pw|\
+	zte,h201l)
+		ucidef_add_switch "switch0" \
+			"0:lan" "1:lan" "2:lan" "3:lan" "4t@eth0"
+		;;
+	bt,homehub-v2b)
+		ucidef_add_switch "switch0" \
+			"1:lan" "2:lan" "3:lan" "4:lan" "5t@eth0"
+		;;
+	netgear,dgn3500|\
+	netgear,dgn3500b)
+		ucidef_add_switch "switch0" \
+			"0:lan:4" "1:lan:3" "2:lan:2" "3:lan:1" "5t@eth0"
+		;;
+	zyxel,p-2601hn)
+		ucidef_add_switch "switch0" \
+			"0:lan" "1:lan" "2:lan" "3:lan" "5t@eth0"
+		;;
+	*)
+		ucidef_set_interface_lan 'eth0'
+		;;
+	esac
+}
+
+lantiq_setup_dsl()
+{
+	local board="$1"
+	local annex="a"
+
+	case "$board" in
+	arcadyan,arv7506pw11|\
+	arcadyan,arv7525pw|\
+	arcadyan,arv752dpw|\
+	arcadyan,arv752dpw22|\
+	arcadyan,arv8539pw22|\
+	avm,fritz7312|\
+	avm,fritz7320|\
+	siemens,gigaset-sx76x|\
+	zte,h201l)
+		annex="b"
+		;;
+	esac
+
+	ls /lib/modules/$(uname -r)/ltq_atm* 1> /dev/null 2>&1 && \
+		ucidef_add_atm_bridge 1 32 "llc" "bridged" "dsl"
+
+	if grep -qE "system type.*: (VR9|xRX200)" /proc/cpuinfo; then
```

Comment:
This should never be true for the xway target which does not support the xrx200

Reply:
@hauke What about xway_legacy, amazonse, falcon?